### PR TITLE
feat: upgrade to sumac

### PR DESCRIPTION
**Pull request by ghassanmas**
v19.0.0 sumac release

### Todos before merge
- [x] Ensure changelog exists for v19
- [x] Ensure CI/Tests are working as expected
- [x] Ensure sumac branch is up-to-date with main (updated as of December 16, 12 PM Pakistan time)
- [x] https://github.com/overhangio/tutor-forum/pull/58 has been merged and changelog has been updated

### Merge Option
- [x] Ensure "Merge Commit" option is used to merge the branch into master (Responsible: @DawoudSheraz ). This is needed to ensure the commit history when master is merged back into nightly

### CODE REVIEW

File: tutorforum/patches/openedx-common-settings
Comment (Ian2012) [inline tutorforum/patches/openedx-common-settings:1]: We need to update the `FORUM_V2_SERVICE_URL` and `COMMENTS_SERVICE_V2_URL` settings to `http|https://{{LMS_HOST}}` in production, or else you would get this error:

`urllib3.exceptions.ReadTimeoutError: HTTPConnectionPool(host='localhost', port=8000): Read timed out. (read timeout=5.0)`
or
`django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'localhost:8000'. You may need to add 'localhost' to ALLOWED_HOSTS`

Comment (regisb) [reply tutorforum/patches/openedx-common-settings:1]: :thinking: No, I don't think that's correct. In forum v2, edx-platform does not communicate with the forum app via HTTP, but via a native HTTP API. Actually, these settings [don't even exist](https://github.com/openedx/edx-platform/pull/35671/files). What makes you think we need to define these settings?

Comment (Ian2012) [reply tutorforum/patches/openedx-common-settings:1]: I was testing this branch locally and on production and it applied this patch to the edx-platform branch:

https://github.com/openedx/edx-platform/compare/open-release/sumac.master...edly-io:edx-platform:forum_v2

This is the line:

https://github.com/overhangio/tutor-forum/pull/49/files#diff-0d2d2c5ee050cc46953dd441dc0d4182751734361acf18294183d2928664418eR29-R31

I can test without that change see how it goes

Comment (regisb) [reply tutorforum/patches/openedx-common-settings:1]: I think you might be pulling an outdated version of that branch. `FORUM_V2_SERVICE_URL` is present neither in the edly/forumv2 nor the edly/forumv2-sumac branches. Please try to rebuild without cache:

     tutor images build --docker-arg=--no-cache-filter=code openedx

Comment (Ian2012) [reply tutorforum/patches/openedx-common-settings:1]: The branch defined in this repo is `forum_v2`, but seems like the right upstream is `forumv2`

**Pull request merged**